### PR TITLE
Document Xiaomi Cloud method is now available to get bindkeys for v4/v5 devices

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Document Xiaomi Cloud method is now available to get bindkeys for v4/v5 devices

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/128394
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Added integration instructions for Xiaomi BLE devices in Home Assistant, supporting MiBeacon, MiScales, and MiFora protocols.
	- Enabled real-time tracking of sensor values to conserve battery life.
	- Included specific instructions for the MiFlora plant sensor, including firmware updates and battery level access conditions.

- **Documentation**
	- Detailed encryption methods and compatibility notes for using custom firmware with Xiaomi devices, including bindkey acquisition methods.

### CODE REVIEW

**Comment by Jc2k** [comment]
Should we remove mention of the cloud token extractor tool or are there cases where it would still be useful?

File: source/_integrations/xiaomi_ble.markdown [hunk 1]
```diff
@@ -37,6 +37,7 @@ This key is called the bindkey or beaconkey.
 
 There are a few ways to obtain a bindkey for your device:
 
+- For v4 and v5 devices, you can provide your Xiaomi Cloud account login credentials for the account to which the device is bound, and Home Assistant will import the appropriate bind key from your account.
```
Comment (c0ffeeca7) [inline source/_integrations/xiaomi_ble.markdown]:
```suggestion
- For v4 and v5 devices, you can provide your Xiaomi Cloud account login credentials for the account to which the device is bound. Home Assistant will import the appropriate bind key from your account.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @bdraco :+1:

**Comment by bdraco** [comment]
Thanks!

**Pull request merged**